earth_enterprise/src/fusion/tools/gee_test/README

```diff
@@ -25,6 +25,7 @@ sudo /opt/google/bin/geconfigureassetroot --new --assetroot  /usr/local/google/g
 
 sudo /opt/google/bin/geselectassetroot --assetroot  /usr/local/google/gevol_test/assets
```

Comment:
Minor: Can we clarify when we're using `/opt/google/` and when `/usr/local/google/`?